core/network/impl/protocols/protocol_fetch_available_data.hpp

```diff
@@ -34,8 +34,9 @@ namespace kagome::network {
                                     host,
                                     make_protocols(kFetchAvailableDataProtocol,
                                                    genesis_hash,
-                                                   "polkadot"),
-                                    log::createLogger(kName)},
+                                                   chain_spec),
```

Comment:
[rust](https://github.com/paritytech/polkadot/blob/3fd990501423edca102bc473c2bb1c2ce9b93c23/node/network/protocol/src/request_response/mod.rs#L277-L286)
```suggestion
                                                  "polkadot"),
```

Reply:
Provided link refers only to polkadot. This case more universal, and allows to serve parachains on kusama too